packages/libs/error-stack/README.md

```diff
@@ -10,7 +10,7 @@
 
 Also check out our [Announcement Post] for `error-stack`!
```

Comment:
'announcement post' should not really be capitalized.

Reply:
Auto-merge was a bit too trigger happy it seems. Tim has fixed #631